sdk/appconfiguration/azure-appconfiguration-provider/README.md

```diff
@@ -110,6 +110,37 @@ Once the provider is refreshed, the configurations can be accessed as normal. An
 
 For additional info check out [Dynamic Refresh](https://learn.microsoft.com/azure/azure-app-configuration/enable-dynamic-configuration-python) on MS Learn.
 
+## Selecting Feature Flags
+
+You can enable loading of feature flags by setting `feature_flags_enabled` to `True`. By default all feature flags with no label are loaded.
```

Comment:
> By default all feature flags with no label are loaded.

This statement is a bit confusing. Does that mean we'll always load feature flags with null label by default, even if `feature_flags_enabled` is not set to true?